server/tests/load_postgres_data.py

```diff
@@ -55,10 +55,12 @@ def main():
         return 1
 
     dbs = ["state", "precomputed", "metadata"]
-    remote_postgres_password = getpass.getpass(
-        prompt=("Password for CloudSQL instance of '%s' environment "
-                "of user '%s': " % (
-                    args.environment, args.remote_postgres_user)))
+    remote_postgres_password = os.environ.get("POSTGRES_PASSWORD", "")
```

Comment:
BTW Postgres has a convention for this: [`PGPASSWORD`](https://www.postgresql.org/docs/current/libpq-envars.html#id-1.7.3.21.3.4.6.1.1).

Reply:
got it, sorry, will fix when got time